microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/config/java/NoValueAssignedToPropertyQuickFix.java

```diff
@@ -117,4 +126,30 @@ private static String getPropertyName(Diagnostic diagnostic, JavaCodeActionConte
 	private static String getTitle(String propertyName, String configFileName) {
 		return MessageFormat.format(CODE_ACTION_LABEL, propertyName, configFileName);
 	}
+
+	/**
+	 * Returns a code action for <code>diagnostic</code> that causes
+	 * <code>item</code> to be added to
+	 * <code>microprofile.tools.validation.unknown.excluded</code> client
+	 * configuration
+	 *
+	 * @param item       the item to add to the client configuration array
+	 * @param diagnostic the diagnostic for the <code>CodeAction</code>
+	 * @return a code action that causes <code>item</code> to be added to
+	 *         <code>microprofile.tools.validation.unknown.excluded</code> client
+	 *         configuration
+	 */
+	private CodeAction createAddToExcludedCodeAction(String item, Diagnostic diagnostic) {
+		CodeAction insertCodeAction = new CodeAction("Exclude '" + item + "' from unassigned property validation?");

```

Comment:
Use `String.format("Exclude '%s' " from unassigned property validation", item)` (question mark unnecessary)

Reply:
Good catch i will fix it

Reply:
I would maybe call it just "Exclude '%s' from property validation". The unassigned part might be a bit confusing to users.

Reply:
fixed, now I use MessageFormat